keycloak-event-emitter/src/main/java/io/cloudtrust/keycloak/eventemitter/EventEmitterProvider.java

```diff
@@ -261,12 +261,11 @@ private ExtendedAdminEvent completeAdminEventAttributes(IdentifiedAdminEvent adm
         String resourcePath = extendedAdminEvent.getResourcePath();
         if (resourcePath != null && resourcePath.startsWith("users")) {
             // parse userID
-            String pattern = ".*([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})";
+            String pattern = "^users/([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})$";
```

Comment:
The location really starts with "users/" at the beginning of the string? Or starting your regexp with ^ was a mistake?

Reply:
Yes, it does start with "users/", thus I've made the regex more precise. Do you see an issue with ^?

Reply:
No, I just wanted to be sure that the **^** was not an error